x-pack/plugins/maps/public/lens/choropleth_chart/expression_renderer.tsx

```diff
@@ -92,6 +93,19 @@ export function getExpressionRenderer(coreSetup: CoreSetup<MapsPluginStartDepend
         handlers.done();
       };
 
+      const dimensionsEvent: DimensionsEvent = {
```

Comment:
Can you explain this a little more? Region map does not display a percentage, but rather any metric aggregation. Not sure I understand setting dimensions for a map.

Reply:
Dimensions here refers to the size of the Lens workspace. The expression renderers can now request ideal dimensions / aspect ratio from the Lens editor. For example, horizontal bar charts can now request a 16:9 ratio.


https://github.com/elastic/kibana/assets/315764/facb0f99-6a05-4dbf-aead-896ca84ca7da

In the case of the chloropleth, we want it to act as it does today (fill all the available space). That's the reason the renderer needs to request 100% for both width and height.

Reply:
something to consider based on this comment and my preference (but of course the decision is up to you Drew) - we use the word 'dimension' in Lens with many meanings. For example, if you search for chart dimension in google you get this answer:

> What is a chart dimension?
> Defining the dimensions of a chart - Support - Bizzdesign Support
> The dimensions of a chart are used to determine what object information will be shown in the chart. Depending on the type of chart this information: is placed along one or more axes (in the bar chart for example)


We call dimension exactly that and I think that we should reconsider if we should also use the same word for chart size. In my opinion we should stick to `chartSize` instead.

Reply:
Sure. Had this same thought when Nathan asked his question 👍

Reply:
Done.